concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1815,8 +1815,12 @@ jobs:
           file: paas-cf/concourse/tasks/smoke-tests-run.yml
 
         ensure:
-          task: remove-temp-user
-          file: paas-cf/concourse/tasks/delete_admin.yml
+          do:
+          - task: get-cf-cli-config
+            file: paas-cf/concourse/tasks/get-cf-cli-config.yml
+
```

Comment:
To my understanding `get-cli-config` was introduced to serve as a common step to enable the remove-healthcheck-db.yml common task in 56c2f5e8482fa4935b5888db1c872b8bfb4bb688.  Do you suggest that as a later change we merge those jobs too?